src/components/profile-item/ProfileItem.tsx

```diff
@@ -8,29 +8,25 @@ import { styles } from '~/components/profile-item/ProfileItem.styles'
 import useBreakpoints from '~/hooks/use-breakpoints'
 import { ProfileItemType } from '~/components/profile-item/complete-profile.constants'
 import { UserRoleEnum } from '~/types'
-import CreateOffer from '~/containers/offer-page/create-offer/CreateOffer'
-import { useDrawer } from '~/hooks/use-drawer'
-import AppDrawer from '~/components/app-drawer/AppDrawer'
 
 interface ProfileItemProps {
   item: ProfileItemType
   isFilled?: boolean
   userRole: UserRoleEnum | ''
+  handleOpenDrawer?: () => void | undefined
 }
 
 const ProfileItem = ({
   item,
   userRole,
-  isFilled = false
+  isFilled = false,
+  handleOpenDrawer = undefined
```

Comment:
we don't have to set `undefined` as a default value for an object property, because it will be `undefined` if the object doesn't contain a property